hpms/src/test/java/gov/cms/ab2d/hpms/service/MockedFetchTest.java

```diff
@@ -38,6 +39,9 @@ class MockedFetchTest {
     @Container
     private static final PostgreSQLContainer postgreSQLContainer = new AB2DPostgresqlContainer();
 
+    @Container
+    private static final AB2DLocalstackContainer localstackContainer = new AB2DLocalstackContainer();
```

Comment:
Is this really needed in all the places it has been added?  HPMS doesn't seem to have a need for using LocalStack.

Reply:
The same reason we have postgres everywhere we need the localstack. It connects to sqs and will fail if there is no environment. If the project uses logmanager it needs the container with it. Once we have sqs up and running we can go back and mabye remove some of the postgres containers.

Reply:
Everything uses events and they need sqs to have that functionality working